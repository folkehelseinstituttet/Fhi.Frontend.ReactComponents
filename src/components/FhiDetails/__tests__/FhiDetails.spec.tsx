import renderer from 'react-test-renderer';
import Details from '../FhiDetails';

describe('FhiDetails', () => {
  test('snapshot test', () => {
    const details = renderer.create(
      <Details
        className="test"
        items={[
          {
            title: 'Dette er en tittel',
            text: 'Dette er beskrivelse',
          },
        ]}
      />,
    );
    const tree = details.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
