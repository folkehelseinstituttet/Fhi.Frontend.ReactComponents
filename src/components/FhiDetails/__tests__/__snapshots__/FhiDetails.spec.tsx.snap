// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiDetails snapshot test 1`] = `
<div
  className="FhiDetails__DetailsContainer-sc-1qzqcal-0 hhjZpM test"
>
  <span
    className="FhiDetails__DetailsTitle-sc-1qzqcal-1 bhIGLJ"
  >
    Dette er en tittel
  </span>
  <span>
    Dette er beskrivelse
  </span>
</div>
`;
